ocaml/idl/datamodel.ml

```diff
@@ -3890,9 +3889,11 @@ module VDI = struct
     ; {
         param_type= DateTime
       ; param_name= "snapshot_time"
-      ; param_doc= "Storage-specific config"
+      ; param_doc=
+          "Storage-specific config. When the timezone is missing, UTC is \
+           assumed"
       ; param_release= tampa_release
-      ; param_default= Some (VDateTime Date.never)
+      ; param_default= Some (VDateTime Date.epoch)
```

Comment:
Is this just a different name but the same underlying date? Epoch is in the past but never could have been in the future.

Reply:
It might be better to capture the intended meaning using `never` as an alias.

Reply:
Date.never is 0. I don't know why None is used for many of these, now using epoch has the same semantic meaning as None in many of these, but I don't see the special treatment, it's just assumed that seeing 1970 means never, which is awkward